### PR TITLE
Codestyle (PHP CS Fixer, PHPStan, Unit-tests Runner)

### PR DESCRIPTION
**Pull request by dankovalenko-paysera**
Here, code style fixes have been made, and most of the errors highlighted by the static analyzer have been corrected (some errors were intentionally added to the baseline, as they are insignificant)

### CODE REVIEW

File: .github/workflows/php-cs-fixer.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup PHP for checking styling
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          coverage: none
+          extensions: mbstring, pdo, xml, xdebug
```
Comment (anatoliyLegkiy) [inline .github/workflows/php-cs-fixer.yml]: Do we need xdebug here?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      xdebug \
+	  @composer
+
+RUN echo "\n[PHP]" >> /usr/local/etc/php/conf.d/docker-fpm.ini  \
+  && echo "error_reporting=E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED" >> /usr/local/etc/php/conf.d/docker-fpm.ini \
+  && echo "memory_limit=512M" >> /usr/local/etc/php/conf.d/docker-fpm.ini \
+  && echo "upload_max_filesize=16M" >> /usr/local/etc/php/conf.d/docker-fpm.ini \
+  && echo "max_post_size=16M" >> /usr/local/etc/php/conf.d/docker-fpm.ini
+
+RUN echo "\n[xdebug]" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini  \
```
Comment (anatoliyLegkiy) [inline Dockerfile]: Do we need xdebug here?

**Review by anatoliyLegkiy — APPROVED** [review]

File: .github/workflows/php-cs-fixer.yml (hunk 1 above)
Comment (dankovalenko-paysera) [reply .github/workflows/php-cs-fixer.yml]: Probably, no

File: Dockerfile (hunk 2 above)
Comment (dankovalenko-paysera) [reply Dockerfile]: Removed

**Pull request merged**